src/traffic/from-traffic.ts

```diff
@@ -69,7 +69,7 @@ export function fromTraffic(
           await delay(entry.time)
         }
 
-        return response
+        return isUniqueHandler ? response.clone() : response
```

Comment:
What are the repercussions of always cloning the response?

Reply:
I have detailed the repercussions of cloning (and not cloning) the response in [this comment in the original issue](https://github.com/mswjs/source/issues/66#issuecomment-2435349671). In brief summary, "cloning" a response does not duplicate the response data in memory, it points to the same internal buffer.

The response data is held in memory until there are no more unread responses. This should be fine since we are already keeping response data in memory for all generated handlers when we call `toResponse`.